packages/holocron/__tests__/holocronModule.spec.jsx

```diff
@@ -474,6 +474,41 @@ describe('holocronModule', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('should pass the moduleState to the module by default, if a reducer and name are specified', () => {
+    const reducer = () => fromJS({ mockKey: 'mockValue' });
+    const Module = holocronModule({
+      name: 'mock-module',
+      reducer,
+    })(({ moduleState }) => <div>{JSON.stringify(moduleState)}|{typeof moduleState}</div>);
+    const mockStore = createStore((state) => state, fromJS({ modules: { 'mock-module': { mockKey: 'mockValue' } } }));
+    const component = renderer.create(
+      <Provider store={mockStore}>
+        <Module />
+      </Provider>
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
Can we use inline snapshots? These are really small

Reply:
Adding inline snapshots caused prettier to re-format this file, i'm investigating why it was ignore our eslint config

Reply:
addressed 8b66203